pkg/controller/metrics/metricscontroller.go

```diff
@@ -20,12 +20,15 @@ This module implements a entry into the OpenSDS metrics controller service.
 package metrics
 
 import (
+	"context"
```

Comment:
Rename this file to pkg/controller/metrics/metrics_controller.go

Reply:
renamed